conda/cli/conda_argparse.py

```diff
@@ -831,7 +831,7 @@ def configure_parser_install(sub_parsers):
 
     Install a specific version of 'python' into an environment, myenv::
 
-        conda install -p path/to/myenv python=3.7.13
+        conda install -p path/to/myenv python=3.10.9
```

Comment:
This feels like updating the year on website footer templates; or remembering to write the correct year on checks during the first few moths. OK though.

Reply:
Agreed, changing..

Reply:
```suggestion
        conda install -p path/to/myenv python=3.10
```